fitnessllm_dataplatform/stream/strava/etl_utils.py

```
@@ -0,0 +1,146 @@
+                           timestamp=timestamp,
+                           status=Status.SUCCESS)
+        except Exception as e:
+            logger.error(f"Error while inserting for {stream_enum.value} for {athlete_id}: {e}")
+            insert_metrics(metrics_list=metrics,
+                           destination=f"dev_strava.metrics",
+                           timestamp=timestamp,
+                           status=Status.FAILURE)
+
+
+
+def insert_metrics(metrics_list: list[Metrics],
```

Ruff (0.8.2)

Missing return type annotation for public function `insert_metrics`

Add return type annotation: `None`

(ANN201)